group-generators/helpers/data-providers/atst/interface-schema.json

```diff
@@ -4,26 +4,55 @@
   "providerClassName": "AttestationStationProvider",
   "functions": [
     {
-      "name": "Get attestation receivers",
+      "name": "Get receivers",
       "functionName": "getAttestations",
       "countFunctionName": "getAttestationsCount",
       "description": "Returns all the receivers of a specific attestation on AttestationStation",
       "args": [
+        {
+          "name": "account",
+          "argName": "account",
```

Comment:
I think here is "creator" (sorry I haven't see this in the first review, my bad)